playbooks/roles/jenkins/tasks/jobs.yml

```diff
@@ -42,7 +42,11 @@
     - '../../../../job_definitions/{{ jobs }}.yml'
 
 - name: Test job definitions before updating
```

Comment:
Can we update the steps to `Test job definitions (as Jenkins user) before updating` (or similar)?